tests/systemd_testsuite/test_01_basic.pm

```diff
@@ -18,7 +18,36 @@ use utils 'zypper_call';
 use power_action_utils 'power_action';
 
 sub run {
-    select_console 'root-console';
+    #prepare test
+    assert_script_run 'cd /var/opt/systemd-tests';
+    assert_script_run './run-tests.sh TEST-01-BASIC --setup 2>&1 | tee /tmp/testsuite.log', 600;
+    assert_script_run 'ls -l /etc/systemd/system/testsuite.service';
```

Comment:
These steps seems to be the same for all test cases, it would make sense to extract it to a function.
Maybe even create a base class, so it also implements the post_fail_hook.

Reply:
Ok